build.gradle.kts

```diff
@@ -58,7 +58,6 @@ allprojects {
 
         dependencies {
             "errorprone"("com.google.errorprone:error_prone_core")
-            "errorprone"("com.jakewharton.nopen:nopen-checker")
```

Comment:
this one was a bit weird for me, happy to bring it back and add the annotations if you like

Reply:
it's errorprone-based DesignForExtension, the abstract classes can be tagged `@Open` and I think things should work